### DISCUSSION TITLE
Cannot use R help by F1 and help panel is blank

### DISCUSSION THREAD

**Discussion by davyzhu** [posit-dev/positron#5510] [Q&A] [answered]
Hi, I installed Positron and RStudio with R 4.4.2.  
On RStudio, I can use R help by F1. 
On Positron, it shows "No R help available". And the help panel is blank. But if I hover on the function, it will show a mini help window.

Any suggestion? Thanks!

![image](https://github.com/user-attachments/assets/7bda3ccf-a282-4a12-ad5e-990a4f455765)
![image](https://github.com/user-attachments/assets/dcc9adfa-3abc-441f-ab72-3fabd91069fb)
![image](https://github.com/user-attachments/assets/2b87fac6-db36-452a-8a2f-1120b9c54ad6)

**Comment by jonvanausdeln** [comment, marked as answer]
@davyzhu , can you provide a little more information about your system.  Can you go to help->about and copy out the version information?

Also, are you trying to get the help inside the editor pane (such as in an R file)?

**Reply by davyzhu** [reply]
Thank you! I restarted [positron](https://github.com/posit-dev/positron), and it worked. 



> @davyzhu , can you provide a little more information about your system. Can you go to help->about and copy out the version information?
> 
> Also, are you trying to get the help inside the editor pane (such as in an R file)?
